src/components/Editor/Editor.css

```diff
@@ -132,8 +132,8 @@ html[dir="rtl"] .editor-mount {
   color: var(--theme-content-color3);
 }
 
-.debug-line .CodeMirror-line {
-  background-color: var(--breakpoint-active-color) !important;
+.debug-line .CodeMirror-linenumber {
+  background-color: transparent !important;
```

Comment:
hmm looks like we might have a specificity war here -- might be something to work on at a later point